taxonomies/ingredients_processing.txt

```diff
@@ -328,7 +331,7 @@ en:concentrated
 de:konzentriert, konzentrierter, konzentrierte, konzentriertes
 # de:doubles:konzentriertes-apfelpüree, apfelsaft-aus-konzentriertem-fruchtsaft
 # de:parsing:before and after, space
-es:concentrado,concentrada
+es:concentrado,concentrada, a partir de concentrado, a partir de concentrada
```

Comment:
This is different. "concentrated something" is a different ingredient than "something from concentrate". We should not mix them.

There is already another entry for it:

en:from-concentrate
es:a base de concentrado,a base de concentrada
fr:à base de concentré, à partir de concentré

you should add "a partir de concentrado, a partir de concentrada" as synonyms of "a base de concentrado".

Reply:
ok